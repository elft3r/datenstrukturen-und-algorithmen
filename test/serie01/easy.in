100
7 -641 899 -1 -1
6 -280 956 -1 1
13 -321 -626 -1 -1
16 600 457 -1 1
3 -628 981 -1 -1
1 108 866 -1 1
6 1 75 1 -1
1 -204 -235 -1 -1
15 313 -577 1 1
7 -363 354 1 -1
9 -222 -474 1 1
15 134 874 -1 1
15 -135 167 1 1
15 474 184 -1 1
5 754 -590 -1 -1
20 138 879 1 1
12 992 -799 1 -1
11 -471 409 1 -1
19 -530 904 -1 1
4 351 -789 1 -1
8 -669 646 -1 1
1 -128 -843 1 1
11 915 -478 1 1
9 -426 505 -1 -1
12 -912 -508 -1 1
4 779 -4 -1 1
20 71 -857 -1 -1
13 -679 -35 1 1
13 -67 -680 -1 1
16 220 972 1 1
7 180 442 1 1
16 795 -870 -1 -1
16 -432 623 -1 -1
4 -424 -457 1 -1
9 -147 -108 1 1
5 470 -835 1 -1
1 981 -619 -1 1
20 -795 388 -1 -1
20 33 651 -1 1
11 -389 81 -1 1
11 -372 161 1 1
4 -489 -29 1 1
10 475 391 1 -1
14 387 783 1 1
14 -876 -75 -1 1
2 905 464 1 -1
5 -300 76 1 1
13 -506 395 -1 1
18 -728 -562 -1 -1
4 -227 -440 1 1
7 82 -463 1 -1
15 -155 91 -1 1
17 183 -538 -1 1
14 668 545 -1 1
9 -436 364 -1 1
10 -265 391 1 1
10 -605 351 -1 1
16 829 -991 1 -1
16 -69 -907 1 1
4 -746 779 -1 1
19 949 -976 1 1
1 643 826 -1 1
2 -632 742 1 -1
13 -142 463 -1 1
14 923 123 1 -1
20 -768 -300 1 1
0 144 -574 1 -1
3 -535 -343 -1 -1
11 -703 -976 1 1
7 -620 -328 -1 -1
6 -614 769 -1 1
20 -465 21 -1 -1
6 -75 520 1 1
10 996 -33 -1 1
8 658 316 1 -1
15 606 -488 1 1
1 -374 567 1 -1
17 -402 -905 1 1
19 510 -32 -1 1
0 -860 -512 1 -1
6 1 -793 -1 -1
19 103 -768 -1 -1
1 -252 895 1 1
14 591 387 1 1
14 888 -894 1 1
20 -847 -676 -1 1
12 800 285 1 1
0 -710 -828 1 1
17 -558 -982 -1 -1
6 786 -604 1 1
2 -728 587 -1 -1
12 628 -455 1 -1
18 -123 891 -1 1
1 873 409 1 -1
17 748 -201 -1 -1
6 -392 551 1 1
18 319 -622 1 1
6 -351 808 -1 -1
15 -338 -267 1 1
9 -448 -548 1 -1
