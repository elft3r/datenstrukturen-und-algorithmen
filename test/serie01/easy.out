899
-9048
-626
-85059
-628
866
1
-235
-233969
354
-20778
482622
50975
-66458
-164
6523413
992
-880
5149144
789
-22263
-843
7783
-426
-8016
1570
786
-105931
-148792
1093564
7186
-870
623
457
-6759
1305
-619
407
-4266042
28604
-6131
-1065
-391
385362
-175833
-441
-520
164899
-728
-1774
-463
113945
-1039807
-49821
21532
12495
-39875
991
-937299
-3829
-1628440
826
1374
128327
-800
-5240508
144
-535
-125529
-328
-9222
444
3785
35679
-342
-69218
567
-1842059
-1451632
-860
1
-768
895
283602
-130134
1031833
112240
-710
1540
-902
141
628
-2498775
409
-547
2448
-1097805
-351
-290296
448
